src/composables/useRenderer/index.ts

```diff
@@ -1,4 +1,4 @@
-import { Color, WebGLRenderer } from 'three'
+import { ACESFilmicToneMapping, Color, WebGLRenderer } from 'three'
```

Comment:
(I can't add a comment there because it wasn't changed in this PR, but ...)

... It looks like the comment on line 80 is now out of sync:

```jsdoc
   * @default NoToneMapping
```

I think it should now be:

```jsdoc
   * @default ACESFilmicToneMapping
```